t/04-openresty/ffi/304-proxy_wasm_set_property_getter.t

```diff
@@ -88,13 +88,16 @@ ok
     }
 --- response_body
 ok
---- error_log eval
+--- match_error_log eval
 [
-    qr/\[info\] .*? \[hostcalls\] on_response_body/,
-    qr/\[info\] .*? wasmx.my_property: my_value/,
+    "[hostcalls] on_response_body",
+    "wasmx.my_property: my_value",
+    "[hostcalls] on_response_body",
+    "wasmx.my_property: my_value",
```

Comment:
Can this be (or be made) accepted as a string? Like:
```
--- match_error_log
[hostcalls] on_response_body
wasmx.my_property: my_value
[hostcalls] on_response_body
wasmx.my_property: my_value
```

Then loaded into an array the same way `error_log` or `grep_error_log_out` can?

Reply:
Possibly, but there's some logic in the test helper for handling repeated tests (using `$repeated_req_idx`) which checks for the input type and that might get in the way. I'll give it a try.